oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -114,6 +114,11 @@ organization's Core Contributors, however, we recognize that circumstances
 differ and a different commitment may be more appropriate. Before signing the
 agreement, please `Contact the Program Administrators`_ to discuss.
 
+The scope of the work or contributions considered as part of the Core 
+Contributors duties, and counted towards an organization's commitments, is
+defined at 
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/3593502844/Core+Contributor+-+Work+Scope>`_
```

Comment:
```suggestion
`<https://openedx.atlassian.net/wiki/spaces/COMM/pages/3593502844/Core+Contributor+-+Work+Scope>`_
```

Reply:
@sarina It looks like I can't modify the branch I had originally created here - and even as a suggestion like here github doesn't let me apply the change. I assume this is due to the permissions changes I've had lately?

Would it be possible to have r/w access on this repo? Besides allowing me to fix this, I would also be able to merge PRs like on this OEP.